<launch>
  <arg name="camera_topic" default="/real_camera"/>
  <arg name="int_camera_id" default="2"/>

    <node name="physical_camera" pkg="robot_driving" type="StitchingScript.py" output="screen">
      <param name="camera_topic" value="$(arg camera_topic)"/>
      <param name="int_camera_id" value="$(arg int_camera_id)"/>
    </node>
</launch>